u5-rest-apis/app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const { json, urlencoded } = require('body-parser')
+
+const { readBooks, readBook, createBook, replaceBook, updateBook, deleteBook } = require('./features/book-controller')
+const { covies, covie, createCovie, updateCovie, deleteCovie } = require('./features/covid-controller')
+
+const app = express()
+
+// Middleware
+app.use(express.static('public'))
+app.use(json())
+app.use(urlencoded({ extended: false }))
+
+// Routes
+app.get('/books', readBooks)
+app.get('/books/:isbn13', readBook)
+app.post('/books', createBook)
+app.put('/books/:isbn13', replaceBook)
+app.patch('/books/:isbn13', updateBook)
+app.delete('/books/:isbn13', deleteBook)
+
+// Routes Lab
+app.get('/covid', covies)
+app.get('/covid/:stateId', covie)
```

Comment:
ฉันคิดว่าคุณควรใส่กริยาในชื่อตัวควบคุมของคุณ เช่น:
I think you should put a verb in your controller's name, ex:
```js
readCovies
readCovie
```
ฉันเองก็ไม่แน่ใจเหมือนกันว่า `covie` เป็นคำ ;) บางที:
I'm also not sure that `covie` is a word ;) Perhaps:
```
readCovidRecords
readCovidRecord
```